src/genie/libs/parser/hp_comware/__init__.py

```diff
@@ -0,0 +1,3 @@
+from genie import abstract
```

Comment:
folder name `hp_comware`, we might want to simplify like just `comware`. I could find kind of version, Comware 5 and Comware 7. They are totally different? if very similar, `comware` works, but if totally different, we might want to use `comware5` and `comware7`.

By the way, are you planning to contribute HPE Comware Unicon plugin as well?

Reply:
I'm already contribute, https://github.com/CiscoTestAutomation/unicon.plugins/pull/9, and the plugin is available in the current release

Reply:
The outputs are slightly different, this parser has been validated for Comware 5.
About the plugin the state machine is the same.

Reply:
Adding a folder to hp_comware as comware5, but changing the name of hp_comware might break the unicon plugin, so maybe change the folder hp_comware to hp and add sub folders as comware5, comware7 and procurve, and at the same time change the unicon folder to hp
@tahigash what do you suggest?

Reply:
sorry, I didn't notice you already contributed the plugin. I will check the folder name in team on Monday and will update you.

Reply:
we talked in our team. and decided to change to `comware` from `hp_comware`.
could you please change to `comware` on this PR? We will change to `comware` in unicon.plugins for 21.2 release.

Reply:
@tahigash but what about versions 5 and 7? I got an HPE Simulator so after build some comware5 plugins i might build some comware7

Reply:
@renatoalmeidaoliveira I was thinking 5 and 7 are similar and can be in same category. or they are different and better to be separated?

Reply:
comware7 `display interface` output
```
GigabitEthernet1/0
Current state: UP
Line protocol state: UP
Description: GigabitEthernet1/0 Interface
Bandwidth: 1000000kbps
Maximum Transmit Unit: 1500
Internet protocol processing: disabled
IP Packet Frame Type:PKTFMT_ETHNT_2, Hardware Address: 5000-0002-0000
IPv6 Packet Frame Type:PKTFMT_ETHNT_2, Hardware Address: 5000-0002-0000
Media type: twisted pair, loopback: not set, promiscuous mode: not set
1000Mb/s, Full-duplex, link type: autonegotiation,
flow-control: disabled
Output queue - Urgent queuing: Size/Length/Discards 0/100/0
Output queue - Protocol queuing: Size/Length/Discards 0/500/0
Output queue - FIFO queuing: Size/Length/Discards 0/75/0
Last clearing of counters: Never
Last 300 seconds input rate: 0.00 bytes/sec, 0 bits/sec, 0.00 packets/sec
Last 300 seconds output rate: 0.00 bytes/sec, 0 bits/sec, 0.00 packets/sec
Input:
  0 packets, 0 bytes
  0 broadcasts, 0 multicasts, 0 pauses
  0 errors, 0 runts, 0 giants
  0 crc, 0 overruns
  0 drops, 0 no buffers
Output:
  0 packets, 0 bytes
  0 broadcasts, 0 multicasts, 0 pauses
  0 errors, 0 underruns, 0 collisions
  0 deferred, 0 lost carriers
```

comware7 `display interface` output

```
GigabitEthernet3/8/0/48 current state: UP
 IP Packet Frame Type: PKTFMT_ETHNT_2, Hardware Address: cc3e-5f69-abcd
 Description: ## TRUNK INTERFACE ##
 Loopback is not set
 Media type is twisted pair
 Port hardware type is  1000_BASE_T
 1000Mbps-speed mode, full-duplex mode
 Link speed type is autonegotiation, link duplex type is autonegotiation
 Flow-control is not enabled
 The Maximum Frame Length is 9216
 Broadcast MAX-ratio: 100%
 Unicast MAX-ratio: 100%
 Multicast MAX-ratio: 100%
 Allow jumbo frame to pass
 PVID: 1
 Mdi type: auto
 Link delay is 0(sec)
 Port link-type: trunk
  VLAN passing  : 1(default vlan), 3, 9, 17, 20, 28, 203, 207, 550
  VLAN permitted: 1(default vlan), 3, 9, 17, 20, 28, 203, 207, 550
  Trunk port encapsulation: IEEE 802.1q
 Port priority: 0
 Last clearing of counters:  Never
 Peak value of input: 4382127 bytes/sec, at 2020-11-03 15:00:46
 Peak value of output: 76824715 bytes/sec, at 2019-11-06 16:58:49
 Last 300 seconds input:  2 packets/sec 1145 bytes/sec 0%
 Last 300 seconds output:  514 packets/sec 542086 bytes/sec 0%
 Input (total):  3184372634 packets, 4508060351380 bytes
         3180540219 unicasts, 0 broadcasts, 3832415 multicasts, 0 pauses
 Input (normal):  3184372634 packets, - bytes
         3180540219 unicasts, 0 broadcasts, 3832415 multicasts, 0 pauses
 Input:  0 input errors, 0 runts, 0 giants, 0 throttles
         0 CRC, 0 frame, - overruns, 0 aborts
         - ignored, - parity errors
 Output (total): 189514390228 packets, 183658868807300 bytes
         161859083717 unicasts, 2279737294 broadcasts, 25375569217 multicasts, 0 pauses
 Output (normal): 189514390228 packets, - bytes
         161859083717 unicasts, 2279737294 broadcasts, 25375569217 multicasts, 0 pauses
 Output: 0 output errors, - underruns, - buffer failures
         0 aborts, 0 deferred, 0 collisions, 0 late collisions
         0 lost carrier, - no carrier
```
They're similar but the changes may be problematic for example the interface operation state that on comware7 apears on another line

Reply:
@tahigash we keep the parser separeted or just comware?

Reply:
looks like very similar. I think we can have only `comware`

Reply:
@tahigash Ok, going to make some changes to support both then.

Reply:
@tahigash folder name changed.

Reply:
confirmed the change